cloudera/hue
# Enabling background --Welcome Tour

## What changes were proposed in this pull request?

- Changes for enabling background in welcome tour

- (Please fill in changes proposed in this fix)

## How was this patch tested?

Go to the help tab browser (left side bar with "?" Icon)

Click on "Welcome Tour"

Welcome screens gets displayed on the screen and the background is hidden.

(Please explain how this patch was tested. Ex: unit tests, manual tests)

Manual testing in the following browsers
Chrome
Safari





- (Please explain how this patch was tested. Ex: unit tests, manual tests)
- (If this patch involves UI changes, please attach a screen-shot; otherwise, remove this)

Before Fix: Background is hidden.

<img width="1685" alt="Screenshot 2022-05-24 at 7 13 47 PM" src="https://user-images.githubusercontent.com/103004488/170050434-f3f4b023-a66b-4978-8e46-806455fd95a1.png">


After Fix: Able to see the background
<img width="1726" alt="Screenshot 2022-05-24 at 6 52 57 PM" src="https://user-images.githubusercontent.com/103004488/170110180-f8be25a9-c3da-4344-ab52-dfb81eccc9af.png">



Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.
